File: 백준/Bronze/10797. 10부제/10부제.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int date = sc.nextInt();

        int violate = 0;
        for (int i = 0; i < 5; i++) {
            int carNum = sc.nextInt();
            
            if (date == carNum) {
                violate++;
            }
        }

        System.out.println(violate);
    }
}

File: 백준/Bronze/10797. 10부제/10부제.cc
#include <iostream>
using namespace std;

int main() {
	
	int date;
	cin >> date;

	int violate = 0;
	for (int i = 0; i < 5; i++)
	{
		int carNum;
		cin >> carNum;

		if (date == carNum)
		{
			violate++;
		}
	}

	cout << violate << endl;
}